data/en/cfinclude.json

```diff
@@ -2,34 +2,31 @@
 	"name":"cfinclude",
 	"type":"tag",
 	"syntax":"<cfinclude template=\"\" runonce=\"true|false\">",
-	"script":"include \"template.cfm\" runonce=true;",
+	"script":"include \"template.cfm\" runonce=true|false;",
 	"returns":"",
 	"related":["cfmodule"],
 	"description":"Includes the content from the referenced file (template). The content may be executed as CFML, see compatibility info below. You can embed cfinclude tags recursively. For another way to encapsulate CFML, see cfmodule. (A CFML page was formerly sometimes called a CFML template or a template.)",
 	"params": [
-		{"name":"template","description":"A logical path to a CFML page.","required":true,"default":"","type":"string","values":[]}
-		, {"name":"runonce","description":"If set to true, the given template (if already processed) is not processed again for a given request. CF10+","required":false,"default":"false","type":"boolean","values":[]}
-
+		{"name":"template","description":"A logical path to a CFML page.","required":true,"default":"","type":"string","values":[]},
+		{"name":"runonce","description":" CF10+ If set to true, the given template is not processed again for a given request if it has already been processed.","required":false,"default":"false","type":"boolean","values":[]}
 	],
 	"engines": {
-		"coldfusion": {"minimum_version":"", "notes":"CF9: Added the script syntax: include.\nCF10: Added runonce attribute.\nCF11: Changed behaviour such that only files with the extension cfm or cfml are compiled and executed by cfinclude (configurable in Application.cfc via this.compileextforinclude), all other files will be statically included.", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-tags/tags-i/cfinclude.html"},
+		"coldfusion": {"minimum_version":"", "notes":"CF11: Changed behaviour such that only files with the extension cfm or cfml are compiled and executed by cfinclude (configurable in Application.cfc via this.compileextforinclude), all other files will be statically included.", "docs":"https://helpx.adobe.com/coldfusion/cfml-reference/coldfusion-tags/tags-i/cfinclude.html"},
```

Comment:
Why are you deleting `CF9: Added the script syntax: include.\nCF10: Added runonce attribute.\n` ?

Reply:
It was hard to read so I set the `CF*+` keys at the corresponding places.

Reply:
Yeah I think the CF9+ was added on the wrong example... I would still like to keep that CF9 added script syntax in the engine notes since I think that is useful, I'm ok with the CF10 one being documented in the attributes.

Reply:
Yeah I changed the position of CF9+.

For notes the line breaks do not work as discussed in #618.

Reply:
It seems that the line breaks are not converted properly. They are replaced with a space instead. Should `paragraphFormat()` used instead or additionally or is there anything else wrong?